packages/flutter_tools/lib/src/ios/device_ios.dart

```diff
@@ -518,12 +518,21 @@ class IOSSimulator extends Device {
     String logFilePath = path.join(
       homeDirectory, 'Library', 'Logs', 'CoreSimulator', simulatorDeviceID, 'system.log'
     );
+
     if (clear)
       runSync(['rm', logFilePath]);
+
+    // TODO: The log message prefix could be shortened or removed.
```

Comment:
I learned *why* Google does this the other day.  It's not necessarily your username, but rather the username of the person who has the most context on the issue. :)  The person to ask about the TODO.